src/plugins/intel_cpu/tests/functional/subgraph_tests/src/remove_reorder.cpp

```diff
@@ -0,0 +1,135 @@
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace CPUTestUtils;
+using namespace InferenceEngine;
+using namespace ov::test;
+
+namespace SubgraphTestsDefinitions {
+
+/*
+  This test runs the following subgraph:
+
+              param1          param2
+                |               |
+              (INT)             |
+                |               |
+             Convert            |
+                |               |
+         (No Reorder BF16)      |
+                |               |
+      const     |   const       |
+           \    |    /          |
+            \   |   /          /
+              Range           /
+                |            /
+                \           /
+                 \         /
+                  \       /
+                    MatMul 
+                      |
+                    Result
+  
+  The main purpose of this test is checking the code path when `Any-Convert-(FP32)-Range` pattern is disabled to transformate `Any-Convert-(BF16)-Range` on the SPR machine.
+  The reason to disable this conversion is `Any-Convert-(BF16)` may cause precision loss if `Any` is a relative big INTEGER value.
+  This scenario is always generated from PT FE or PPP. In order to avaid conversion loss, just keep this pattern and ignore automatic `EnforceInferencePrecision`.
+*/
+
+using RemoveUselessReorderCPUTestParams = std::tuple<
+    ElementType,
+    std::vector<InputShape>,
+    std::vector<ov::Shape>
+>;
+
+class RemoveUselessReorderCPUTest: public testing::WithParamInterface<RemoveUselessReorderCPUTestParams>,
+                                 virtual public SubgraphBaseTest {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<RemoveUselessReorderCPUTestParams> obj) {
+        ElementType inType;
+        std::vector<InputShape> inputShapes;
+        std::vector<ov::Shape> targetShapes;
+        std::tie(inType, inputShapes, targetShapes) = obj.param;
+        std::ostringstream result;
+        result << "IS=";
+        for (const auto& shape : inputShapes) {
+            result << ov::test::utils::partialShape2str({shape.first}) << "_";
+        }
+        result << "TS=";
+        for (const auto& shape : inputShapes) {
+            result << "(";
+            if (!shape.second.empty()) {
+                for (const auto& itr : shape.second) {
+                    result << ov::test::utils::vec2str(itr);
+                }
+            }
+            result << ")";
+        }
+        result << "Prc=" << inType;
+        return result.str();
+    }
+    void SetUp() override {
+        targetDevice = ov::test::utils::DEVICE_CPU;
+        ElementType inType;
+        std::vector<InputShape> inputShapes;
+        std::vector<ov::Shape> targetShapes;
+        std::tie(inType, inputShapes, targetShapes) = this->GetParam();
+
+        if (inType == ElementType::bf16) {
```

Comment:
Please use additional_config preperty to initialize inference_precision: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/mvn.cpp#L116

Reply:
use additional_config style.